test/inductor/test_ck_backend.py

```diff
@@ -13,6 +13,12 @@
 from torch.testing._internal.inductor_utils import HAS_CPU, HAS_CUDA
 
 
+try:
+    from .test_fp8 import _quantize_rowwise, _quantize_tensorwise
+except ImportError:
+    from test_fp8 import _quantize_rowwise, _quantize_tensorwise
```

Comment:
Wondering when ```from .test_fp8 import ...``` would fail.

Reply:
I've seen a similar pattern in a number of tests under `test/inductor` - I believe it's something related to how tests are internally set up